contracts/mockContracts/MockDGXDex.sol

```diff
@@ -0,0 +1,187 @@
+pragma solidity ^0.4.18;
+
+//ask:
+//dst address change?
+//should init another token outside in test?
+
+// blocknumber + 5 >= current block number ? revert
+// signatures - skip for now.
+
+import "../ERC20Interface.sol";
+import "../Withdrawable.sol";
+import "./ecverify.sol";
+
+
+/// @title Mock Digix DEX
+/// @author Tal Baneth
+/// @dev a dummy contract that simulates Digix contract for purchase/sell with verified signatures. 
+contract MockDGXDEX is Withdrawable {
+    uint  constant internal G_TO_MG = (10**3);
```

Comment:
can u add some documentation what this means?
or maybe can be clearer names?

Reply:
changed to GRAM_TO_MILLIGRAM.